cmd/containerd/command/main.go

```diff
@@ -183,10 +183,10 @@ can be used and modified as necessary as a custom configuration.`
 			err error
 		}
 
-		// run server initialization in a goroutine so we don't end up blocking important things like SIGTERM handling
+		// run server initialization in a goroutine, so we don't end up blocking important things like SIGTERM handling
 		// while the server is initializing.
 		// As an example opening the bolt database will block forever if another containerd is already running and containerd
-		// will have to be be `kill -9`'ed to recover.
+		// will have to be `kill -9` to recover.
```

Comment:
I think the `'ed` here was intentional to describe that "containerd has to be killed" (using `9` / `SIGKILL`), but it's a bit awkward indeed.

Perhaps alternatively, this could be written something like;

```
As an example, opening the bolt database blocks forever if a containerd instance
is already running, which must then be forcibly terminated (SIGKILL) to recover.
```